server/controllers/previewCtrl.js

```diff
@@ -173,9 +177,9 @@ async function renderMemoPreviewPage(req, res, next) {
     const memoDataAsArray = []
     // will be used later for add flag outdated to new pm
     memoDataAsArray.push(apiMemoData)
+    // const { semester, memoCommonLangAbbr } = apiMemoData
+    // const memoLangAbbr = memoCommonLangAbbr || userLang
     applicationStore.memoData = markOutdatedMemoDatas(memoDataAsArray, miniLadokObj)[0]
-    const { semester, memoCommonLangAbbr } = apiMemoData
-    const memoLangAbbr = memoCommonLangAbbr || userLang
```

Comment:
I'm not up to date with the memoLang vs userLang issue right now, but should this really be removed now? Or maybe this is fixed in the other PR. I will look into that after lunch.